frontend/src/components/object/ObjectTable.vue

```diff
@@ -85,18 +96,47 @@ async function showPermissions(objectId: string) {
 
 watch(getObjects, async () => {
   clearSelected();
+  loadLazyData(lazyParams);
+});
 
-  // Filter object cache to this specific bucket
-  const objs: Array<COMSObjectDataSource> = getObjects.value.filter(
-    (x: COMSObject) => x.bucketId === props.bucketId
-  ) as COMSObjectDataSource[];
+onMounted(() => {
+  loading.value = true;
 
-  tableData.value = objs.map((x: COMSObjectDataSource) => {
-    x.lastUpdatedDate = x.updatedAt ?? x.createdAt;
-    return x;
-  });
+  lazyParams.value = {
+    first: 0,
+    rows: 10,
+    sortField: null,
+    sortOrder: null,
+    filters: filters.value
+  };
+  loadLazyData(lazyParams);
```

Comment:
same here. no need to pass the lazyParams because they are defined 'globally' in the component's state `lazyParams.value`.

Reply:
I made it optional param